SQL/view_job_spec.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE job
+ADD job_spec_summary TEXT, job_url TEXT;
```

Comment:
does url column need to be this large?

Reply:
Used varchar 255 originally but one of the URLs got truncated so I upped it to TEXT